ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -142,7 +142,7 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
      * If a cell is not refined in the hierarchy, its weight is set to its
      * volume.  If a cell is refined, its weight is set to zero.
      */
-    SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > getCellWeightVariable() const;
+    SAMRAI::pdat::CellVariable<NDIM, double>* getCellWeightVariable() const;
```

Comment:
Like Aaron I don't see an advantage of this design: this is always going to return a non-null pointer so it should just be a reference.

Reply:
I needed to revert this change anyway --- we need to provide a `Pointer` here because ultimately these weights are used in `SAMRAIVectorReal`, which expects a `Pointer`.